include/RevProc.h

```diff
@@ -153,13 +153,21 @@ class RevProc{
   void MarkLoadComplete(const MemReq& req);
 
   ///< RevProc: Get pointer to Load / Store queue used to track memory operations
-  std::shared_ptr<std::unordered_map<uint64_t, MemReq>> GetLSQueue(){ return LSQueue; }
+  auto GetLSQueue() const { return LSQueue; }
```

Comment:
prefer the explicit declaration of the return type (I know it's long, but nice to be explicit about what it is)

Reply:
I didn't do it for shorter names, but for single-points-of-definition (of the type of `LSQueue`). Since `GetLSQueue()` is a simple accessor which just returns `LSQueue`, we shouldn't need to repeat the type.